bazel/repositories.bzl

```diff
@@ -231,6 +233,11 @@ def _envoy_api_deps():
     )
 
 def envoy_dependencies(path = "@envoy_deps//", skip_targets = []):
+
+    skip_or_blacklist = depset() + skip_targets
+    if "//bazel:linux_ppc":
```

Comment:
Yeah, this will always evaluate `True` today, since the semantics are the same as Python, i.e. `True` unless empty/`None`.

Reply:
I'm not sure it's possible to use Bazel's built-in config settings at this point in the dependency resolution. You may need to do something messy to get this working until the Lua dependency can be Bazelified with the new C++ external deps work.

Maybe like this?

```
def _is_linux_ppc():
  if ctx.os.name != "linux":
    return False
  res = ctx.execute(["uname", "-p"])
  return "ppc" in res.stdout
```